### ISSUE TITLE
LinkedIn Deprecated Sign In with LinkedIn V1

### ISSUE THREAD

**Issue by charlower** [auth0/passport-linkedin-oauth2#97]
### Description

https://learn.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin

Looks like they only offer Sign in with Linked In OpenID now, so I don't think this strategy works anymore. More info here:

https://learn.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin-v2

I'm not smart enough to fix it.

Edit: Strategy won't work for NEW LinkedIn apps.

**Comment by nguaman** [comment]
I found several people with the same problem:
Here is the thread from the official Auth0 forum.
https://community.auth0.com/t/use-linkedin-social-connection-with-basic-profile-on/111588/4

Apparently, the generated URL is as follows:

`https://www.linkedin.com/oauth/v2/authorization?login_hint=&prompt=login&response_type=code&redirect_uri=https%3A%2F%2Fxxxx.eu.auth0.com%2Flogin%2Fcallback&scope=r_liteprofile&state=xxxx&client_id=xxxx
`

In the scope, it still uses '**r_liteprofile**', which should be replaced with '**profile**' from what I understand.

**Comment by charlower** [comment]
yeah i tried using profile but was getting an 'internal oauth' error.

**Comment by mahaboobkhan29** [comment]
Any solution how to use singin with linkedin in our website?
